docs/user/BuildWithWSL.md

```diff
@@ -0,0 +1,38 @@
+# Build with WSL
+Windows Subsystem for Linux, or WSL for short is a way for you to mount a Linux-based OS onto your Windows machine, allowing you to build OpenROAD-flow-scripts both locally and via Docker. 
+
+## Install WSL
+Instructions to install WSL can be found [here](https://learn.microsoft.com/en-us/windows/wsl/install). You may use any kernel supported, such as: CentOS 7, Ubuntu 20.04, Ubuntu 22.04, RHEL 8, Debian 10 and Debian 11.
+
+We recommend that users follow the Docker build by continuing onto the guide below. However, if you wish to install locally, you may follow the build locally instructions [here](./BuildLocally.md).
+
+> **TIP**: You may delete your WSL kernel using this [guide](https://pureinfotech.com/remove-linux-distro-wsl/).
+
+
+## Docker configuration
+This section assumes you have set up Docker for Windows already. If not, please refer to the Docker official website for instructions [here](https://docs.Docker.com/desktop/install/windows-install/).
+
+You need to enable the following option to allow WSL to use Docker. 
+
+> Insert pic 1 here
+
+> Insert pic 2 here
```

Comment:
When are we planning on filling these place holders?

Reply:
@vvbandeira Inserted the images.